python/sglang/srt/entrypoints/openai/serving_chat.py

```diff
@@ -826,8 +841,47 @@ def _process_tool_calls(
         tools: List[Any],
         tool_call_parser: Optional[str],
         finish_reason: Dict[str, Any],
+        tool_choice: Optional[Union[str, Any]] = None,
     ) -> tuple[Optional[List[ToolCall]], str, Dict[str, Any]]:
```

Comment:
maybe have this as a NamedTuple? It became hard to tell what the this three fields mean.

Reply:
Friendly ping.